### ISSUE TITLE
Holding a potted plant makes you unknown over radio (and I guess speech in general) 

### ISSUE THREAD

**Issue by MrMelbert** [tgstation/tgstation#86829]
### Issue Summary:

Holding a potted plant makes anyone unable to determine your voice even though that doesn't make sense

### Reproduction Steps:

- Pick up plant
- Talk on radio
- You are now unknown, something normally people have to pay TC to accomplish

**Comment by Metekillot** [comment]
We not wanting to keep this?

**Comment by MrMelbert** [comment]
Not really, having unknown speech is a powerful ability (as mentioned something people normally have to pay TC for) to have lying around the station

**Comment by Metekillot** [comment]
Mimicking speech I can understand being powerful but unknown speech just screams suspicious to me. I don't see the advantage but I'm working on pulling apart voice returning to fix it anyway

**Comment by Loafin34** [comment]
i believe this is because of the potted plant's unexaminable trait, which is shared with the infil suit. maybe like...... tone it down or something if it gets fixed
